README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
```

Comment:
I think we should add a more descriptive text to our users here. "Press the space bar to begin" what exactly? Also some informative text about happens when the user presses the space bare after a game has started would be nice here as well.